src/Core/System/Country/Aggregate/CountryStateTranslation/CountryStateTranslationCollection.php

```diff
@@ -11,6 +11,9 @@
 #[Package('fundamentals@discovery')]
 class CountryStateTranslationCollection extends EntityCollection
 {
+    /**
+     * @return array<array-key, string>
```

Comment:
should be this for entities
```suggestion
     * @return array<string, string>
```